parsers/ShortcodeParser.php

```diff
@@ -612,4 +619,51 @@ function ($matches) use ($tags, $parser) {
 
 		return $content;
 	}
+
+	/**
+	 * Embed shortcode parser from Oembed. This is a temporary workaround.
+	 * Oembed class has been replaced with the Embed external service.
+	 *
+	 * @param $arguments
+	 * @param $content
+	 * @param $parser
+	 * @param $shortcode
+	 * @param array $extra
+	 *
+	 * @return string
+	 */
+	public static function handle_embed_shortcode($arguments, $content, $parser, $shortcode, $extra = array()) {
```

Comment:
This should be in its own class, e.g. `EmbedShortcodeProvider`. Don't pollute the ShortcodeParser core with oEmbed-specific things.